packages/prop-controllers/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@makeswift/prop-controllers",
+  "version": "0.0.1",
+  "license": "MIT",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/index.cjs",
+  "types": "./dist/types/index.d.ts",
```

Comment:
Does this give us the type setup we want for IDEs? For example, does jump to definition work? Do we need to add source maps?

Reply:
Let's make sure to try it from an external package like Leo. The easiest way is using the `file://` protocol from pnpn for a dependency

Reply:
Didn't know there was a `file://` protocol. I tested it and it's working.

https://github.com/makeswift/makeswift/assets/13066728/c2d00706-15fe-4333-b6f1-afd9b2acf27c